boards/phytec/phyboard_electra/doc/index.rst

```diff
@@ -0,0 +1,143 @@
+The board configuration also enables support for the semihosting debugging console.
+
+See the `PHYTEC AM64x Product Page`_ for details.
+
+.. figure:: img/phyCORE-AM64x_Electra_frontside.webp
+   :align: center
+   :alt: phyBOARD-Electra AM64x
+
+   PHYTEC phyBOARD-Electra with the phyCORE-AM64x SoM
+
+Hardware
+********
+The AM64x phyBOARD-Electra kit features the AM64x SoC, which is composed of a
+dual Cortex-A53 cluster and two dual Cortex-R5F cores in the MAIN domain as
+well as a single Cortex-M4 core in the MCU domain. Zephyr is ported to run on
+the M4F core and the following listed hardware specifications are used:
+
+- Low-power ARM Cortex-M4F
+- Memory
+
+   - 256KB of SRAM
+   - 2GB of DDR4
+
+- Debug
+
+   - XDS110 based JTAG
+
+Supported Features
+==================
+
+The phyboard_electra/am6442/m4 configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| PINCTRL   | on-chip    | pinctrl                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by the port.
+
+Devices
+========
+System Clock
+------------
+
+This board configuration uses a system clock frequency of 400 MHz.
+
+DDR RAM
+-------
+
+The board has 2GB of DDR RAM available. This board configuration
+allocates Zephyr 4kB of RAM (only for resource table: 0xa4100000 to 0xa4100400).
+
+Serial Port
+-----------
+
+This board configuration uses a single serial communication channel with the
+MCU domain UART (MCU_UART0).
+
+GPIO
+----
+
+The phyCORE-AM64x has a heartbeat LED connected to gpio6. It's configured
+to build and run the `basic/blinky` sample.
+
+SD Card
+*******
+
+Download PHYTEC's official `WIC`_ as well as `BMAP`_ and flash the WIC file with
+an etching software onto an SD-card. This will boot Linux on the A53 application
+cores of the SoM. These cores will then load the zephyr binary on the M4 core
+using remoteproc.
+
+The default configuration can be found in
+:zephyr_file:`boards/phytec/phyboard_electra/phyboard_electra_am6442_m4_defconfig`
+
+Flashing
+********
+
+The Linux running on the A53 uses the remoteproc framework to manage the M4F co-processor.
+Therefore, the testing requires the binary to be copied to the SD card to allow the A53 cores to
```

Comment:
It would be nice to provide instructions to do the same from u-boot console, that provides a faster development environment. I do something like below for development on m4/a53 on AM62 SK
```
dhcp; setenv serverip 192.168.0.108; rproc init; rproc stop 0; tftp
0x82000000 zephyr.elf; rproc load 0 0x82000000 $filesize;  rproc start
0;tftp 0x82000000 zephyr.bin; dcache flush; icache flush; dcache off;
icache off; go 0x82000000
```

Reply:
Thanks for sharing those commands with me! We currently figuring out how to improve Zephyr/MCU+ integration into our products and they will be very helpful.

I will update documentation for both products later. Kinda thinking this might be a nice uboot environment function...